src/bundle/SystemInfo/Exception/SysInfoServiceNotFoundException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Bundle\SystemInfo\SystemInfo\Exception;
+
+use Exception;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException as BaseNotFoundException;
+
+class SysInfoServiceNotFoundException extends BaseNotFoundException
```

Comment:
```suggestion
class ServiceNotFoundException extends BaseNotFoundException
```

We know that it's System Info related from namespace.

Reply:
Well, I introduced this prefix only to avoid conflicts and increase readability in here: https://github.com/ibexa/system-info/pull/4/files#diff-a716c6769d2cec068ae7e9abff4f2e4e9c0833ebe899b2a0b545e582c935408fR35.

Reply:
Indeed, I've overlooked this.